definitions/types/azure-security.json

```diff
@@ -1,7 +1,13 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
   "type": "object",
-  "title": "Azure Security information",
-  "description": "Informs downstream services of ports to open for communication",
-  "additionalProperties": false
-}
+  "title": "Security",
+  "description": "Azure Security Configuration",
+  "additionalProperties": false,
+  "required": [],
+  "properties": {
+    "iam": {
+      "$ref": "./gcp-security-iam.json"
```

Comment:
```suggestion
      "$ref": "./azure-security-iam.json"
```

Reply:
Good catch! you can see I copy-pasta'd your stuff!